kotlinx-coroutines-core/jvm/src/debug/ArtificialStackFrames.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.debug
+import kotlinx.coroutines.*
+
+/**
+ * A collection of artificial stack trace elements to be included in stack traces by the coroutines machinery.
```

Comment:
We can slightly improve this documentation: mention that it is for debugging purposes only and leave links to to the debug mode document and the corresponding property